network/src/peer/mod.rs

```diff
@@ -420,7 +445,23 @@ where
                     error_msg,
                 );
             }
-            NetworkMessage::RpcRequest(_) | NetworkMessage::RpcResponse(_) => {
+            NetworkMessage::RpcRequest(request) => {
+                if let Err(err) = self
+                    .inbound_rpcs
+                    .handle_inbound_request(&mut self.peer_notifs_tx, request)
+                    .await
+                {
+                    warn!(
+                        NetworkSchema::new(&self.network_context)
+                            .connection_metadata(&self.connection_metadata),
+                        error = %err,
+                        "{} Error handling inbound rpc request: {}",
+                        self.network_context,
+                        err
+                    );
+                }
+            }
+            NetworkMessage::RpcResponse(_) => {
                 let notif = PeerNotification::NewMessage(message);
                 self.rpc_notifs_tx.send(notif).await.map_err(|err| {
```

Comment:
Interesting that we used to notify new inbound RPC responses even on requests...  Either that or we just had a poorly commented notification channel.